python/src/aiconfig/editor/server/server.py

```diff
@@ -109,27 +110,88 @@ def _get_server_state(app: Flask) -> ServerState:
 
 @app.route("/")
 def home():
-    ss = _get_server_state(app)
-    ss.count += 1
-    LOGGER.info("Count: %s", ss.count)
     return app.send_static_file("index.html")
 
 
-@app.route("/test")
-def test():
-    return {"key": 6}
+@app.route("/api/load", methods=["POST"])
+def load():
+    ss = _get_server_state(app)
```

Comment:
minor nit: having some meaningful variable names makes it easy to read the code down the line (maybe rename to `state`?)